internal/credential/store_test.go

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright The ORAS Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package credential
+
+import (
+	"os"
+	"reflect"
+	"testing"
+)
+
+func TestNewStoreError(t *testing.T) {
+	tmpDir := t.TempDir()
+	filename := tmpDir + "testfile.txt"
```

Comment:
It is safer to use [`filepath.Join()`](https://pkg.go.dev/path/filepath#Join) instead of concatenation.

Reply:
```suggestion
	filename := path.Join(tmpDir, "testfile.txt")
```